kotlin-yaml/src/test/kotlin/com/baeldung/yaml/kaml/KamlUnitTest.kt

```diff
@@ -0,0 +1,342 @@
+package com.baeldung.yaml.kaml
+
+import com.baeldung.yaml.verifyUsers
+import com.baeldung.yaml.yamlkt.YamlktUnitTest
+import com.charleskorn.kaml.*
+import kotlinx.serialization.decodeFromString
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+import kotlin.test.assertFalse
+
+class KamlUnitTest {
+
+    private val yamlDefault = Yaml.default
```

Comment:
Let's use a fully qualified import to make it clear which library we are importing from

Reply:
Fixed